pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -773,6 +773,19 @@ def test_aggregate_mixed_types():
     tm.assert_frame_equal(result, expected)
 
 
+def test_aggregate_categorical_lost_index():
+    # GH: 28641
```

Comment:
can you add a few words on what the relevant bug was

Reply:
Done